index.html

```diff
@@ -36,6 +36,196 @@ <h1>
+                    <li>Back End Dev</li>
+                    <li><img src="./assets/Counter.svg" alt=""></li>
+                    <li>2015</li>
+                </ul>
+                <div class="projectSummary">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <div class="projectTechnology">
+                    <span class="tag">HTML</span>
+                    <span class="tag">CSS</span>
+                    <span class="tag">JavaScript</span>
+                </div>
+                <div class="projectDemo">
+                    <button class="enabled">See project</button>
+                </div>
+            </div>
+        </div>
+    </section>
     
+    <section id="aboutme">
+        <div class="top">
+            <span class="heading">About Myself</span>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            <a class="letconnect" href="">LET’S CONNECT</a> 
+            <ul class="socialIconsGrid">
+                <li><a href="#"><img src="./assets/twitter.svg" alt="twitter"></a></li>
+                <li><a href="#"><img src="./assets/linkedin.svg" alt="linkedin"></a></li>
+                <li><a href="#"><img src="./assets/medium.svg" alt="medium"></a></li>
+                <li><a href="#"><img src="./assets/github.svg" alt="github"></a></li>
+                <li><a href="#"><img src="./assets/victory.svg" alt="victory"></a></li>
+            </ul>
+            <div class="action"><button class="enabled">Get my resume</button></div>
+        </div>
+
+        <div class="myskills">
+            <div class="languages">
+                <div class="lan_head">
+                    <div class="lan_heading">
+                        <h5>Languages</h5>
+                    </div>
+                    <a class="dropdown_icon">
+                        <span class="material-symbols-outlined">
+                            expand_more
+                    </span>
+                    </a>
+					</div>
+            <ul class="lan_list">
+							<li class="lan_list_detail">
+								<img class="list_img" src="./assets/javascript_icon.svg" alt="javascript">
+									<p>Javascript</p>
+							</li>
+							<li class="lan_list_detail">
+								<img class="list_img" src="./assets/html_icon.svg" alt="html">
+								<p>HTML</p></li>
+							<li class="lan_list_detail">
+								<img class="list_img" src="./assets/css_icon.svg" alt="css">
+								<p>CSS</p></li>
+						</ul>
+				</div>
+				<div class="languages">
+					<div class="lan_head">
+						<div class="lan_heading">
+							<h5>Frameworks</h5>
+						</div>
+						<a class="dropdown_icon">
+							<span class="material-symbols-outlined">
+								expand_more
+								</span>
+						</a>
```

Comment:
- [ ] As suggested by previous reviewer and demanded by project requirements, kindly use a nested unordered list to list out technologies in the about section. The list should in its basic form look like;
```html
<ul>
  <li>Languges
    <ul>
      <li>HTML</li>
      <li>CSS</li>
      <li>Javascript</li>
    </ul>
  </li>
  <li>Languges</li>
  <li>Skills</li>
</ul>
```